source/map.rst

```diff
@@ -0,0 +1,245 @@
+to either of these approaches to populate a structured grid representation of
+the geometry.
+
+Attila implements the discrete ordinates method using a finite element
+approach on a tetrahedral mesh, allowing the mesh to conform to arbitrarily
+complex shapes of the geometry and for each mesh element to contain a single
+material.  Attila is able to directly import CAD-based geometry in order to
+generate mesh for its transport algorithm.
+
+Frequently, the CAD models available for a given system have been generated
+without engineering analysis in mind.  As such, these models include details
+that are unnecessary for nuclear analysis as well as overlaps of different
+components.  The cleaning and repair of CAD-based geometries is currently not
+automatable and benefits greatly from experience working with previous models
+for this purpose.
+
+Automated Variance Reduction Techniques
+----------------------------------------
+
+As indicated above, Monte Carlo radiation transport can be challenged by
+problems that involve deep penetration of thick shields, a characteristic of
+most shielding problems.  A number of techniques, generally known as variance
+reduction techniques, exist to accelerate these calculations for a specific
+response.  However, as the problems become more complex, it becomes difficult
+to manually configure those techniques for maximum benefit.  Over the last
+15-20 years, a number of approaches have been developed to automate the
+configuration of variance reduction.  Some approaches (MCNP weight window
+generator, MAGIC) use iteration, where one round of Monte Carlo simulation is
+used to improve a guess for the variance reduction parameters for the
+successive round.
+
+An alternative approach is to use the results of a deterministic calculation
+to provide the parameters for variance reduction.  This is effective because
+the deterministic calculation is generally much faster than a Monte Carlo
+calculation with uniform quality of results.  ORNL has implemented the CADIS
+and FW-CADIS methodologies in the tool ADVANTG for this purpose, when used in
+conjunction with MCNP.  CADIS will optimize variance reduction parameters for
+a single response while FW-CADIS can be employed to optimize variance
+reduction parameters for many responses, including global solutions.
+
+Types of Analysis
+------------------
+
+These tools are used in various combinations to accomplish different types of
+analysis.  Depending on the combination of tools, different skills and
+expertise are necessary.
+
+Radiation transport, damage, heating, and tritium breeding
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+A basic radiation transport calculation can be configured to provide a variety
+of nuclear responses in addition to the neutron flux distribution.  These
+responses must be linear combinations of the neutron fluxes, typically based
+on cross-sections or similar nuclear data.  It is common to estimate radiation
```

Comment:
It is common to estimate radiation damage in terms of displacements per atom, H and He gas production, as well as heating from neutrons and other secondary particles.